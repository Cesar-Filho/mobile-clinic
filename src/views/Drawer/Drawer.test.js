import React from 'react';
import renderer from 'react-test-renderer';

import { Drawer } from '.';

describe('<Drawer />', () => {
  it('renderizar sem quebrar', () => {
    const rendered = renderer.create(<Drawer />).toJSON();
    expect(rendered).toMatchSnapshot();
  });
});
