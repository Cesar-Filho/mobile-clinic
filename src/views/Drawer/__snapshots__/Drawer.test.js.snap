// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Drawer /> renderizar sem quebrar 1`] = `
<View
  style={
    Array [
      Object {
        "display": "flex",
        "flexBasis": 0,
        "flexDirection": "column",
        "flexGrow": 1,
        "flexShrink": 1,
        "paddingBottom": 40,
        "paddingLeft": 12,
        "paddingRight": 24,
        "paddingTop": 40,
      },
    ]
  }
>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "borderLeftColor": "#733dbe",
        "borderLeftWidth": 4,
        "borderRadius": 5,
        "elevation": 1,
        "height": 40,
        "justifyContent": "center",
        "marginBottom": 10,
        "opacity": 1,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#733dbe",
            "fontSize": 16,
            "marginLeft": 12,
          },
        ]
      }
    >
      Hello! :3
    </Text>
  </View>
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "borderLeftColor": "#733dbe",
        "borderLeftWidth": 4,
        "borderRadius": 5,
        "elevation": 1,
        "height": 40,
        "justifyContent": "center",
        "marginBottom": 10,
        "opacity": 1,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#733dbe",
            "fontSize": 16,
            "marginLeft": 12,
          },
        ]
      }
    >
      Sair :: bye bye!
    </Text>
  </View>
</View>
`;
